Automattic/jetpack
# AI Assistant: Fix form popover after changing viewport

Follow-up to #32242

## Proposed changes:

* Fixes the popover after changing viewport to mobile and then back to desktop (like when changing portrait to landscape on mobile)

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Add a form block and change the viewport to mobile
* Open the input and check that the input is visible and fixed
* Close the input
* Change to landscape
* Open the input again and check that it takes the width of the form

https://github.com/Automattic/jetpack/assets/8486249/f90e35fd-09a7-49cf-aee6-3573bfbdcdfc
